src/routes/stores.ts

```diff
@@ -7,4 +7,4 @@ export const cdnUrl = (filename: string, dimension: string | null = null) => {
     return `${PUBLIC_CDN_URL}/lyntr/${filename}${dimension ? '_' + dimension : ''}.webp?v=${v}`
 }
 
-export const unreadMessages = writable(0);
+export const unreadMessages = writable(-1);
```

Comment:
```suggestion
export const unreadMessages = writable(0);
```
...???
Would make more sense being 0, but if that's needed then Ok.

Reply:
Oh that's where it is I can't remember why but it works so I don't think I'll bother changing it